### DISCUSSION TITLE
vscode基础配置：vscode基础使用指南 - WxW

### DISCUSSION THREAD

**Discussion by giscus** [Whbbit1999/wxw-blog-comments#3] [General]
# vscode基础配置：vscode基础使用指南 - WxW

https://code.visualstudio.com/https://code.visualstudio.com/  为什么是 vscodevscode 是微软开源的免费的文本编辑器，被广泛地在前端工作中使用。它提供了丰富的插件供人们进行代码编写和文字工作。 可以满足我们平时的工作和学习，最吸引人的是它完全免费。 基础配置我们可以进行一些简单的配置来让 vscode 更加好用。在 setti

https://www.whbbit.cn/wiki/vscode/index.html

**Comment by Whbbit1999** [comment]
vscode配置是很丰富的
